packages/web3-eth-accounts/src/index.js

```diff
@@ -272,20 +297,117 @@ function _validateTransactionForSigning(tx) {
         );
     }
 
-    if (!tx.gas && !tx.gasLimit) {
+    if (
+        (!tx.gas && !tx.gasLimit) &&
+        (!tx.maxPriorityFeePerGas && !tx.maxFeePerGas)
+    ) {
         return new Error('"gas" is missing');
     }
 
-    if (tx.nonce < 0 ||
-        tx.gas < 0 ||
-        tx.gasPrice < 0 ||
-        tx.chainId < 0) {
+    if (tx.gas && tx.gasPrice) {
+        if (tx.gas < 0 || tx.gasPrice < 0) {
+            return new Error('Gas, gasPrice, nonce or chainId is lower than 0');
```

Comment:
```suggestion
            return new Error('Gas or gasPrice is lower than 0');
```

Reply:
I wanted to change these, but technically it's a breaking change since we'll be returning a different error message to the user

Reply:
It's not breaking the api or changing the behaviour, its still returning an error, so it would be caught with a `.catch()`. In general its not a great practice to be filtering errors based on exact matches...

I think in this case we would consider this to be a bug, since its now a misleading error message. Just my two cents.